README.md

```diff
@@ -37,6 +37,17 @@ conda install -c conda-forge kedro
 
 Our [Get Started guide](https://docs.kedro.org/en/stable/get_started/install.html) contains full installation instructions, and includes how to set up Python virtual environments.
 
+### Installation from source
+To access the latest Kedro version before its official release, install it from the source code. First, clone the repository:
+```
+git clone git@github.com:kedro-org/kedro.git
+```
+
+Then change to the `kedro` directory and enter the following:
+```
+pip install .
+```
```

Comment:
What about

```
pip install git+https://github.com/kedro-org/kedro
```

?

Reply:
(To avoid the intermediate cloning step)

Reply:
sounds good